Functions/xASL_bids_CompareStructures.m

```diff
@@ -136,15 +136,17 @@
 function reportTable = xASL_bids_CompareStructures_AddEntriesToTable(reportTable,results,datasetA,datasetB)
 
     % Go through missing folders
-    if size(results.(datasetA).missingFolders,1)>0 && ~isempty(results.(datasetA).missingFolders{1})
+    if numel(results.(datasetA).missingFolders)>0 && ...
+            ~isempty(results.(datasetA).missingFolders{1})
         for iElement=1:size(results.(datasetA).missingFolders,1)
             dataset = cellstr(datasetA);
             name = cellstr('Missing folder');
             message = cellstr(results.(datasetA).missingFolders{iElement,1});
             reportTable = xASL_bids_CompareStructures_AddTableRow(reportTable,dataset,name,message);
         end
     end
-    if size(results.(datasetB).missingFolders,1)>0 && ~isempty(results.(datasetB).missingFolders{1})
+    if numel(results.(datasetB).missingFolders)>0 && ...
```

Comment:
And the same here - it was good replacing size,1 and size,2 with numel, but then this needs to be shadowed on the other lines of code.

Reply:
Fixed here: 4d00c8f 👍